getting-started-guides/on-prem-cluster/standalone-python.md

```diff
@@ -97,15 +97,20 @@ ${SPARK_HOME}/bin/spark-submit \
     --master spark://$HOSTNAME:7077 \
     --executor-memory 32G \
     --jars ${SPARK_JARS},${JAR_RAPIDS}\
+    --conf spark.executor.resource.gpu.vendor=nvidia.com \
```

Comment:
gpu.vendor is only required in Kubernetes, I think.

Reply:
Updated.